public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,142 @@
+block includes
+  include ../_util-fns
+
+:marked
+  # Trying Angular
+
+  Live coding [Quickstart](../quickstart.html) in the browser is a great way to explore Angular.
+  Links on almost every documentation page will open completed samples in the browser where you can
+  play with the code, share your changes with friends, and download and run the code on your own machine.
+
+  As much fun as this is
+  * you can't ship your app in plunker
+  * you aren't always online when writing code
+  * transpiling TypeScript in the browser is slow
+  * the type support, refactoring, and code completion only work in your local IDE

```

Comment:
I know, but I think our beloved tech writer leads may disagree :)
@kapunahelewong - any opinion about this?